database/procedures/clickAnalytics.sql

```diff
@@ -0,0 +1,43 @@
+USE `labby`;
+
+DROP PROCEDURE IF EXISTS `createClickAnalytics`;
+DROP PROCEDURE IF EXISTS `update_count`;
+DROP PROCEDURE IF EXISTS `initialize_count`;
+
+DELIMITER $$
+
+CREATE PROCEDURE `createClickAnalytics` ()
+BEGIN
+
+CREATE TABLE `click_analytics` (
```

Comment:
Let's put the create table call inside the init folder, so it's separate from the stored procedures!